client/my-sites/email/email-management/home/email-plan-mailboxes-list.jsx

```diff
@@ -24,12 +25,27 @@ import Gridicon from 'calypso/components/gridicon';
 import { recordTracksEvent } from 'calypso/state/analytics/actions';
 import { resendVerificationEmail } from 'calypso/state/email-forwarding/actions';
 
-const MailboxListHeader = ( { children } ) => {
+const getListHeaderTextForAccountType = ( accountType, translate ) => {
+	if ( accountType === EMAIL_ACCOUNT_TYPE_FORWARD ) {
+		return translate( 'Email forwards', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Active Email Forwards' )` *ES Score: 8*